File: QuickSort/QuickSort.js
function quickSort(arr) {
    if (arr.length == 0)  return []
    let pivot = arr[arr.length - 1]
    let smaller = []
    let larger = []
    for (item of arr) {
        if (item > pivot) larger.push(item)
        else if (item < pivot) smaller.push(item)
    }
    return quickSort(smaller).concat(pivot).concat(quickSort(larger))
}

File: QuickSort/QuickSort.py
def quickSort(arr):
    if not arr: return []
    smaller, larger = [], []
    pivot = arr[len(arr) - 1]
    for n in arr[:-1]:
        if n >= pivot: larger.append(n)
        elif n < pivot: smaller.append(n)
    return quickSort(smaller) + [pivot] + quickSort(larger)